Puppetfile.lock

```diff
@@ -36,12 +36,12 @@ FORGE
       puppetlabs-apache (< 3.0.0, >= 1.0.0)
       puppetlabs-stdlib (< 5.0.0, >= 4.6.0)
       theforeman-foreman (< 9.0.0, >= 7.0.0)
-    katello-pulp (5.0.1)
+    katello-pulp (5.3.0)
```

Comment:
To keep the changes minimal I'd rather do a 5.0.2 release with that single change. Is it only about that one or are there others?

Reply:
Just this change. A 5.0.2 release would be great! :1st_place_medal:

Reply:
https://forge.puppet.com/katello/pulp/5.0.2/readme